inst/safetyGraphics_app/modules/renderSettings/renderSettings.R

```diff
@@ -49,8 +49,8 @@ renderSettings <- function(input, output, session, data, settings, status){
 
   ns <- session$ns
 
-  charts<-as.vector(filter(chartsMetadata, chart %in% all_charts)[["chart"]])
-  labels<-as.vector(filter(chartsMetadata, chart %in% all_charts)[["label"]])
+  charts<-as.vector(chartsMetadata[["chart"]])
```

Comment:
i dont remember doing this...

Reply:
Heh, that slipped in there as we added ```charts``` as an argument in ```safetyGraphicsApp()```. The user-specified ```charts``` becomes ```all_charts``` in global.R